packages/maskbook/src/extension/background-script/WelcomeServices/restoreBackup.ts

```diff
@@ -48,14 +48,14 @@ export async function restoreBackup(json: object, whoAmI?: ProfileIdentifier) {
                         )
                     }
                 }
-
-                for (const x of data.wallets) {
```

Comment:
Move the restore wallet logic out of personal db consistent check since create wallet db transaction makes current  `FullPersonaDBTransaction` inactive, which no longer accept requests, otherwise it would throw error: 
`DOMException:TransactionInactiveError: Failed to execute 'openCursor' on 'IDBObjectStore': The transaction is not active.(-1)`